src/tasks/cache/bundle.rs

```diff
@@ -72,7 +55,7 @@ impl BundlePoller {
             .error_for_status()?
             .json()
             .await
-            .map(|resp: TxPoolBundleResponse| resp.bundles)
+            .map(|resp: TxCacheBundlesResponse| resp.bundles)
```

Comment:
using the sdk types is blocked until https://github.com/init4tech/tx-pool-webservice/pull/73/files is in